python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -501,16 +504,25 @@ def disk_usage(self):
         """Not yet implemented"""
         raise NotImplementedError
 
-    def inspect(self, x: Optional[str] = None) -> Builder:
+    def inspect(
+            self,
+            x: Optional[str] = None,
+            bootstrap: Optional[bool] = False,
+        ) -> Builder:
         """Returns a builder instance from the name.
 
         Parameters:
             x: If `None` (the default), returns the current builder. If a string is provided,
                 the builder that has this name is returned.
-
+            bootstrap: If set to True, ensure builder has booted before inspecting.
+                
         # Returns
             A `python_on_whales.Builder` object.
         """
+        if bootstrap:
```

Comment:
Note that running the `docker buildx inspect --boostrap [NAME] ` command is necessary, when the bootstrap flag is set to True, as the command modifies the targeted builder state.

Reply:
That seems reasonable